types/datum.go

```diff
@@ -1409,8 +1409,8 @@ func (d *Datum) ToInt64(sc *stmtctx.StatementContext) (int64, error) {
 }
 
 func (d *Datum) toSignedInteger(sc *stmtctx.StatementContext, tp byte) (int64, error) {
-	lowerBound := SignedLowerBound[tp]
-	upperBound := SignedUpperBound[tp]
+	lowerBound := ByteToSignedLowerBound(tp)
```

Comment:
Could you remove the origin definition of SignedLowerBound?

Reply:
@ngaut Do you mean delete `SignedLowerBound` from `types/convert.go`? I remember I tried to search it around and it seems that `SignedLowerBound` used in multiple places. Do you think I should replace them all or?

Reply:
@liufuyang 
I think we should replace them all, so we can completely resolve this issue.